index.html

```diff
@@ -65,7 +65,7 @@ <h3>Instructions</h3>
         </p>
         <br />
         <p>
-          Layers itself are then seperated via <code>:</code>, starting with the
+          Layers itself are then separated via <code>:</code>, starting with the
```

Comment:
This is odd wording. 
```suggestion
          Layers are separated via <code>:</code>, starting with the
```